### PR TITLE
Fixes two bugs in ElementInspector.js and two bugs in LinkInspector.js

### LINKED ISSUE #482: absTime parameter must be reset when the link type changes in linkInspector.js
For example, when you switch from evolving and/or to NBD, the absTime param should clear

### LINKED ISSUE #509: For stochastic UD functions the value is wrong
image

### LINKED ISSUE #510: Increase/decrease are still grayed out after you select a different value

### LINKED ISSUE #511: When you select a link type it clears the text of the intention's func type and sat value
checkCellText probably needs to be updated

### PR DESCRIPTION
**Pull request by i-m-foster**
Closes #509 , #482 , #510 , and #511

### CODE REVIEW

File: leaf-ui/rappid-extensions/ElementInspector.js [hunk 1]
```diff
@@ -314,7 +313,7 @@ var ElementInspector = Backbone.View.extend({
         this.validityCheck(event);
 
         if (this.intention.get('evolvingFunction') != null) {
-            var functionType = this.intention.get('evolvingFunction').get('type');
+            var functionType = this.intention.get('evolvingFunction').get('type');
```
Comment (YesugenB) [inline leaf-ui/rappid-extensions/ElementInspector.js]: Delete the tab space here?

File: leaf-ui/rappid-extensions/LinkInspector.js [hunk 2]
```diff
@@ -127,20 +127,26 @@ var LinkInspector = Backbone.View.extend({
      * Switches from Evolving Relationship to Constant Relationship
      */
     renderConstant: function() {
+        console.log(this.link.get('absTime'));
```
Comment (amgrubb) [inline leaf-ui/rappid-extensions/LinkInspector.js]: Remove console log?

File: leaf-ui/rappid-extensions/LinkInspector.js [hunk 3]
```diff
@@ -127,20 +127,26 @@ var LinkInspector = Backbone.View.extend({
...
+        this.link.set('absTime', null);
         this.link.set('evolving', false);
         this.$el.html(_.template(this.constanttemplate)());
         $('#constant-links').val(this.link.get('linkType'));
         this.setValues($('#constant-links').val(), null, false);
         this.checkCellText();
+        console.log(this.link.get('absTime'));
```
Comment (amgrubb) [inline leaf-ui/rappid-extensions/LinkInspector.js]: Remove console log?

Comment (amgrubb) [reply leaf-ui/rappid-extensions/LinkInspector.js]: There are at least two more in this file.

Comment (amgrubb) [reply leaf-ui/rappid-extensions/LinkInspector.js]: There are at least two more in this file.

**Review by amgrubb — APPROVED** [review]
Did we ever figure out why this PR has so many commits?
Please remove console.log's were appropriate.

**Pull request merged**